test/UnitTest/Microsoft.AspNet.OData.Test.Shared/OpenComplexTypeTest.cs

```diff
@@ -368,6 +548,8 @@ public class OpenCustomer
 
     public class OpenAddress
     {
+        internal const int DefaultValue_Fee = 20;
```

Comment:
>DefaultValue_Fee = 20; [](start = 27, length = 22)

why add in OpenAddress class? maybe it's better in the LineDetails class?

Reply:
I was thinking that DefaultValue_Fee is used by OpenAddress object's ctor to set the value of LineDetails.Fee, therefore it is kind of belong to OpenAddress class. But now think it probably makes more sense to co-locate this with UninitializedValue_Fee.
Moved.


---
In reply to: [190752494](https://github.com/OData/WebApi/pull/1445#discussion_r190752494) [](ancestors = 190752494)